ansible_collections/arista/avd/roles/eos_designs/templates/base/base.py

```diff
@@ -0,0 +1,559 @@
+
+    @cached_property
+    def management_interfaces(self):
+        mgmt_interface = get(self._switch_data, 'mgmt_interface')
+        if (
+                mgmt_interface is not None and
+                self._mgmt_ip is not None and
+                self._mgmt_interface_vrf is not None and
+                self._mgmt_gateway is not None
+        ):
+            return {
+                mgmt_interface: {
+                    'description': 'oob_management',
+                    'shutdown': False, 'vrf': self._mgmt_interface_vrf,
+                    'ip_address': self._mgmt_ip, 'gateway': self._mgmt_gateway,
+                    'type': 'oob'
+                }
+            }
+        return None
+
+    @cached_property
+    def tcam_profile(self):
+        tcam_profile = get(self._platform_settings, 'tcam_profile')
+        if tcam_profile is not None:
+            return {'system': tcam_profile}
+        return None
+
+    @cached_property
+    def platform(self):
+        lag_hardware_only = get(self._platform_settings, 'lag_hardware_only')
+        platform = {}
+        if lag_hardware_only is not None:
+            platform.update({'sand': {'lag': {'hardware_only': lag_hardware_only}}})
+        if (
+                get(self._platform_settings, 'trident_forwarding_table_partition') is not None and
+                get(self._switch_data, 'evpn_multicast') is True
+        ):
+            platform.update({'trident': {'forwarding_table_partition':
+                            get(self._platform_settings, 'trident_forwarding_table_partition')}})
+        if platform:
+            return platform
+        return None
+
+    @cached_property
+    def mac_address_table(self):
+        mac_address_table = get(self._hostvars, 'mac_address_table')
+        if mac_address_table is not None:
+            aging_time = get(mac_address_table, 'aging_time')
+            if aging_time is not None:
+                return {'aging_time': aging_time}
+        return None
+
+    @cached_property
+    def queue_monitor_streaming(self):
+        queue_monitor_streaming = get(self._hostvars, 'queue_monitor_streaming')
+        if queue_monitor_streaming is not None:
+            enable = get(queue_monitor_streaming, 'enable')
+            vrf = get(queue_monitor_streaming, 'vrf')
+            if enable is True or vrf is not None:
+                queue_monitor = {}
+                if enable is True:
+                    queue_monitor.update({'enable': enable})
+                if vrf is not None:
+                    queue_monitor.update({'vrf': vrf})
+                return queue_monitor
+        return None
+
+    @cached_property
+    def management_api_http(self):
+        management_eapi = get(self._hostvars, 'management_eapi')
+        if management_eapi is not None:
+            management_api_http = {'enable_vrfs': {self._mgmt_interface_vrf: {}}}
+            if get(management_eapi, 'enable_http') is not None:
```

Comment:
We are not adding anything here, so we could consider using `management_api_http = management_eapi.fromkeys(['enable_http','enable_https','default_services'])` and then update with the enable_vrfs.